raiden/transfer/events.py

```diff
@@ -124,6 +125,40 @@ def __repr__(self) -> str:
         )
 
 
+@dataclass(frozen=True)
+class ContractSendChannelCoopSettle(ContractSendEvent):
+    """Event emitted if node wants to withdraw from current channel balance."""
+
+    canonical_identifier: CanonicalIdentifier
+    our_total_withdraw: WithdrawAmount
+    partner_total_withdraw: WithdrawAmount
+    expiration: BlockExpiration
+    # This are the partner's signatures of the two withdraws.
+    # Our signature of that data will be constructed again in the event-handler
+    signature_our_withdraw: Signature
+    signature_partner_withdraw: Signature
+
+    @property
+    def channel_identifier(self) -> ChannelID:
+        return self.canonical_identifier.channel_identifier
+
+    @property
+    def token_network_address(self) -> TokenNetworkAddress:
+        return self.canonical_identifier.token_network_address
+
+    def __repr__(self) -> str:
+        return (
+            f"{self.__class__.__name__}< "
+            f"canonical_identifier: {self.canonical_identifier} "
+            f"our_total_withdraw: {self.our_total_withdraw} "
+            f"parner_total_withdraw: {self.our_total_withdraw} "
+            f"expiration: {self.expiration} "
+            f"partner_signature_withdraw: {to_hex(self.signature_partner_withdraw)} "
+            f"partner_signature_our_withdraw: {to_hex(self.signature_our_withdraw)} "
```

Comment:
I'd prefer to use field names as they are, instead of renaming them for the `__repr__` output.

Reply:
Changed to original field names.